src/controllers/shared.controllers/director.residential.address.search.controller.ts

```diff
@@ -114,13 +114,13 @@ export const postDirectorResidentialAddressSearch = async (req: Request, res: Re
 };
 
 const setUpdateBoolean = async (req: Request, isUpdate: boolean, session: Session, officerFiling : OfficerFiling) => {
```

Comment:
Could we have a generic method like this which is called in all controllers?

Reply:
There are two  (non exported) versions of this setUpdateBoolean as they change different flags (CA and HA). The other updates mix in changes to the officer filing so are just done inline.